.gitignore

```diff
@@ -4,3 +4,4 @@ src/config.json
 DS_STORE
 .DS_Store
 src/__generated__
+config.js
```

Comment:
what's config.js? It's src/config.json

Reply:
Actually I couldn't figure out how to import variable from .json file, so i use config as js file.

Reply:
aren't we doing this in Map.jsx?